src/workloads/VSSetup.props

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
You can remove this xml section as it's not needed by msbuild and we usually don't have it in our msbuild files in the repo.